src/resources/vendorlist.json

```diff
@@ -0,0 +1 @@
[3]},{"id":473,"name":"Sub2 Technologies Ltd","policyUrl":"http://www.sub2tech.com/privacy-policy/","purposeIds":[3,4,5],"legIntPurposeIds":[1,2],"featureIds":[]},{"id":467,"name":"Haensel AMS GmbH","policyUrl":"https://haensel-ams.com/data-privacy/","purposeIds":[3,5],"legIntPurposeIds":[],"featureIds":[2]},{"id":488,"name":"Opinary GmbH","policyUrl":"https://opinary.com/privacy-policy/","purposeIds":[],"legIntPurposeIds":[1,2,3,4,5],"featureIds":[]},{"id":490,"name":"PLAYGROUND XYZ EMEA LTD","policyUrl":"https://playground.xyz/privacy/privacy-policy.pdf","purposeIds":[1,3,5],"legIntPurposeIds":[],"featureIds":[]},{"id":464,"name":"Oracle AddThis","policyUrl":"http://www.addthis.com/privacy/privacy-policy/","purposeIds":[1,2,3,4],"legIntPurposeIds":[5],"featureIds":[2]},{"id":491,"name":"Triboo Data Analytics","policyUrl":"https://www.shinystat.com/it/informativa_sito.html","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[]},{"id":499,"name":"PurposeLab, LLC","policyUrl":"https://purposelab.com/privacy/","purposeIds":[5],"legIntPurposeIds":[],"featureIds":[]},{"id":502,"name":"NEXD","policyUrl":"https://nexd.com/privacy-policy","purposeIds":[3,5],"legIntPurposeIds":[1,2,4],"featureIds":[3]},{"id":465,"name":"Schibsted Product and Tech UK","policyUrl":"https://www.schibsted.com/","deletedDate":"2019-07-26T00:00:00Z","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[1]},{"id":497,"name":"Little Big Data sp.z.o.o.","policyUrl":"http://dtxngr.com/legal/","purposeIds":[1,2,4],"legIntPurposeIds":[],"featureIds":[1,2,3]},{"id":492,"name":"LotaData, Inc.","policyUrl":"https://lotadata.com/privacy_policy","purposeIds":[],"legIntPurposeIds":[1,5],"featureIds":[1]},{"id":508,"name":"Lucid Holdings, LLC","policyUrl":"https://luc.id/gdpr","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[1,2,3]},{"id":512,"name":"PubNative GmbH","policyUrl":"https://pubnative.net/privacy-notice/","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[3]},{"id":471,"name":"FlexOffers.com, LLC","policyUrl":"https://www.flexoffers.com/privacy-policy/","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[1,2,3]},{"id":494,"name":"Cablato Limited","policyUrl":"https://cablato.com/privacy","purposeIds":[1,2,4,5],"legIntPurposeIds":[],"featureIds":[]},{"id":516,"name":"Pexi B.V.","policyUrl":"https://pexi.nl/privacy-policy/","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[]},{"id":507,"name":"AdsWizz Inc.","policyUrl":"https://www.adswizz.com/our-privacy-policy/","purposeIds":[],"legIntPurposeIds":[1,2,3,5],"featureIds":[1,2,3]},{"id":482,"name":"UberMedia, Inc.","policyUrl":"http://ubermedia.com/company/products-privacy-policy/#ADDITIONAL_NOTICE","purposeIds":[1,2,3,4,5],"legIntPurposeIds":[],"featureIds":[]},{"id":505,"name":"Shopalyst Inc","policyUrl":"https://www.shortlyst.com/eu/privacy_terms.html","purposeIds":[1,2],"legIntPurposeIds":[],"featureIds":[]},{"id":517,"name":"SunMedia ","policyUrl":"https://www.sunmedia
```

Comment:
This is now dynamic in Frontend, maybe we should think about how to synch up with that one?